back/app/controllers/web_api/v1/reset_password_controller.rb

```diff
@@ -1,47 +1,40 @@
-  class WebApi::V1::ResetPasswordController < ::ApplicationController
-
-    skip_after_action :verify_authorized
-
-
-    def reset_password_email
-      @user = User.not_invited.find_by_cimail params[:user][:email]
-      if !@user
-        # throw active record not found error
-        User.not_invited.find_by! email: params[:user][:email].downcase 
-      end
-      
-      token = ResetPasswordService.new.generate_reset_password_token @user
-      @user.update!(reset_password_token: token)
-      ResetPasswordService.new.log_password_reset_to_segment(@user, token)
-      head :accepted
+class WebApi::V1::ResetPasswordController < ::ApplicationController
+  skip_after_action :verify_authorized
+  skip_before_action :authenticate_user
+
+  def reset_password_email
+    @user = User.not_invited.find_by_cimail params[:user][:email]
+    if !@user
+      # throw active record not found error
+      User.not_invited.find_by! email: params[:user][:email].downcase 
     end
-
-    def reset_password
-      @user = User.not_invited
-        .find_by(reset_password_token: reset_password_params[:token])
-      if @user && ResetPasswordService.new.token_valid?(@user, reset_password_params[:token])
-        if @user.update(password: reset_password_params[:password], reset_password_token: nil)
-          render json: WebApi::V1::UserSerializer.new(
-            @user, 
-            params: fastjson_params
-            ).serialized_json
-        else
-          render json: {errors: @user.errors.details}, status: :unprocessable_entity
-        end
+    
+    token = ResetPasswordService.new.generate_reset_password_token @user
+    @user.update!(reset_password_token: token)
+    ResetPasswordService.new.log_password_reset_to_segment(@user, token)
```

Comment:
Yes, I'm not merging with master all the time because that would give me a lot of work in time. I'm planning to merge this evening or tomorrow, it's going to be quite a challenge to solve all conflicts 😅